### PR TITLE
Avoid copying msg when extracting the event type

### PR DESCRIPTION
**Pull request by skonto**
This is an improvement for #688 .

### CODE REVIEW

**Comment by skonto** [comment]
lint issue is unrelated and it is fixed in #709

File: pkg/channel/consolidated/dispatcher/consumer_message_handler.go [hunk 1]
```diff
@@ -76,29 +75,21 @@ func (c consumerMessageHandler) Handle(ctx context.Context, consumerMessage *sar
...
 	dispatchExecutionInfo, err := c.dispatcher.DispatchMessageWithRetries(
 		ctx,
-		bufferedMessage,
+		message,
 		nil,
 		c.sub.Subscriber,
 		c.sub.Reply,
 		c.sub.DeadLetter,
 		c.sub.RetryConfig,
+		[]binding.Transformer{&te}...,
```
Comment (slinkydeveloper) [inline pkg/channel/consolidated/dispatcher/consumer_message_handler.go:86]: why that? you can just use `&te`

**Comment by skonto** [comment]
@lionelvillard could you add ok to test?

**Comment by skonto** [comment]
@slinkydeveloper gentle ping.

**Pull request merged**